proto/panacea/datadeal/v2/consent.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package panacea.datadeal.v2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+import "gogoproto/gogo.proto";
+import "panacea/datadeal/v2/deal.proto";
+
+// Certificate defines a certificate with signature
+message Certificate {
+  UnsignedCertificate unsigned_certificate = 1;
+  bytes signature = 2;
+}
+
+// UnsignedCertificate defines a certificate information
+message UnsignedCertificate {
+    string cid = 1;
+    string operator_address = 2;
```

Comment:
why don't we use `oracle_address` to be clear?
I think we have many similar words can be confused: node, oracle, operator, validator, etc.
For oracle, it would be better to unify with the word `oracle`.

Reply:
Yes, We used a lot of words for Oracle's address. 
We set this name to `operator_address` in the architecture discussion.
But I think `oracle_address` is better. :)

Reply:
fixed: https://github.com/medibloc/panacea-core/pull/515/commits/8eccb41513d2d18b87ff343c73155def0844c388